pkg/networkservice/mechanisms/vxlan/server.go

```diff
@@ -42,9 +42,9 @@ type vxlanServer struct {
 }
 
 // NewServer - returns a new server for the vxlan remote mechanism
-func NewServer(vppConn api.Connection, tunnelIP net.IP) networkservice.NetworkServiceServer {
+func NewServer(vppConn api.Connection, tunnelIP net.IP, vniOptions ...vni.Option) networkservice.NetworkServiceServer {
```

Comment:
Could we please make the Option pattern here local to this package... importing another packages option here precludes other unrelated Options in the future.